### PR TITLE
Add and remove method

### PR DESCRIPTION
**Pull request by Mikelobi**
### To Do List Structure

- Use webpack to bundle JavaScript.
- Use proper ES6 syntax.
- Configure and setup webpack for the project.
- Add todo item to list and save the item to localStorage.
- Edit todo item and update localStorage with the edited item.
- Delete all completed todo items when the clear all completed button is clicked.

### CODE REVIEW

File: src/edit-task.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export default (arr, index) => {
+  arr[index].description = document.getElementById(`update${index}`).value;
+  return arr;
+};
```
Comment (thecodechaser) [inline src/edit-task.js:4]: - [ ] I am not able to edit the task description, After editing a task if I refresh the page I get the same description as it was before editing. Please make sure editing a task also update the local storage so if a user refreshes the page after editing a task description he should get the edited version of it.

File: src/remove-task.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  static deleteOne(arr, index) {
+    arr.splice(index, 1);
+    for (let i = 0; i < arr.length; i += 1) {
+      arr[i].index = i + 1;
+    }
+    return arr;
+  }
+
+  // Delete all completed tasks
+}
```
Comment (thecodechaser) [inline src/remove-task.js:12]: - [ ] I can not see any option to delete a task from the list. Please make sure there is an option for the user to delete a task from the list, You can add a trash icon that can be used to delete a task from the list. 
  - Please also keep in mind `Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1)`

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Mikelobi👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/edit-task.js (hunk 1 above)
Comment (Mikelobi) [reply src/edit-task.js:4]: @thecodechaser, you can check again by double click on the added description. Edit and click to save. 

edit function key

File: src/remove-task.js (hunk 2 above)
Comment (Mikelobi) [reply src/remove-task.js:12]: @thecodechaser, you can click on the delete icon and check again.
delete function key

**Comment by thecodechaser** [comment]
@Mikelobi You can go for another review from your dashboard

**Review by Deepakdanger — APPROVED** [review]
Hi @Mikelobi, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**